packages/samples/headless-commerce-ssr-remix/app/components/triggers/redirection-trigger.tsx

```diff
@@ -1,19 +0,0 @@
-import {useRedirectionTrigger} from '@/lib/commerce-engine';
```

Comment:
As clarified in the reference docs, the `RedirectionTrigger` controller should only be used exceptionally. I think it's better to remove it from the samples altogether to avoid confusion.